### PR TITLE
Tooltip implementation using i18n_title

### PR DESCRIPTION
**Pull request by tricopterY**
jBox Tooltip implementation using i18n_title, instead of populating jBox: Content - reduces DOM element counts

### CODE REVIEW

**Comment by skaman82** [comment]
very good idea

File: js/gui.js [hunk 1]
```diff
@@ -286,9 +286,10 @@ GUI_control.prototype.content_ready = function (callback) {
         log(this); // ...print out "this", which now refers to each ".cf_tip" DOM element
     });
 
+    //Note: using i18n_title, so no need for convoluted content: $(this).children('.cf_tooltiptext')
     $('.cf_tip').each(function() {
         $(this).jBox('Tooltip', {
-            content: $(this).children('.cf_tooltiptext'),
+           // content: $(this).children('.cf_tooltiptext'),
```
Comment (hydra) [inline js/gui.js]: delete unused code rather than commenting it out.  We can use git history if we ever want it back.

File: tabs/failsafe.js [hunk 2]
```diff
@@ -129,8 +129,7 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
                             <div style="width:60%; float:left;">\
                                 <span>' + channelNames[i] + '</span>\
                             </div>\
-                            <div class="cf_tip" style="width:25%; float:left;">\
-                                <div class="cf_tooltiptext" style="display:none;"><strong>Auto</strong> means Roll, Pitch and Yaw to center and Throttle low. <strong>Hold</strong> means maintain the last good value received.</div>\
+                            <div class="cf_tip" style="width:25%; float:left;" title="' + chrome.i18n.getMessage("failsafeChannelFallbackSettingsAuto") + '">\
```
Comment (hydra) [inline tabs/failsafe.js]: please replace inline styles with CSS.

File: tabs/failsafe.js [hunk 3]
```diff
@@ -146,8 +145,7 @@ TABS.failsafe.initialize = function (callback, scrollPosition) {
                                 <span class="channelname">' + chrome.i18n.getMessage("controlAxisAux" + (aux_index++)) + '</span>\
                                 ' + auxAssignment[aux_assignment_index++] + '\
                             </div>\
-                            <div class="cf_tip" style="width:25%; float:left;">\
-                                <div class="cf_tooltiptext" style="display:none;"><strong>Hold</strong> means maintain the last good value received. <strong>Set</strong> means the value given here will be used. </div>\
+                            <div class="cf_tip" style="width:25%; float:left;" title="' + chrome.i18n.getMessage("failsafeChannelFallbackSettingsHold") + '">\
```
Comment (hydra) [inline tabs/failsafe.js]: please replace inline styles with CSS.

**Comment by tricopterY** [comment]
@hydra - please try again. 
Thanks for the guidance.

**Comment by tricopterY** [comment]
Created a new branch and closed this one.

**Pull request closed without merging**